vnext/src/Libraries/Components/View/ReactNativeViewAttributes.windows.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+'use strict';
+import ReactNativeStyleAttributes from './ReactNativeStyleAttributes';
+
+const UIView = {
+  pointerEvents: true,
+  accessible: true,
+  accessibilityActions: true,
+  accessibilityLabel: true,
+  accessibilityLiveRegion: true,
+  accessibilityRole: true,
+  accessibilityState: true,
+  accessibilityValue: true,
+  accessibilityHint: true,
+  importantForAccessibility: true,
+  nativeID: true,
+  testID: true,
+  renderToHardwareTextureAndroid: true,
+  shouldRasterizeIOS: true,
+  onLayout: true,
+  onAccessibilityAction: true,
+  onAccessibilityTap: true,
+  onMagicTap: true,
+  onAccessibilityEscape: true,
+  collapsable: true,
+  needsOffscreenAlphaCompositing: true,
+  style: ReactNativeStyleAttributes,
+  // [Windows
```

Comment:
Can we add these props upstream?

Reply:
I think the API should be revisited before we upstream these. I think the upstream APIs are ripe for refactoring to something more inline with W3C pointer APIs (e.g., `onTouchStart` -> `onPointerDown`, `onMouseEnter` -> `onPointerEnter`, etc.)

Reply:
We can't fix this in this PR anyway.

Reply:
See comment above, but upstream does seem to already have `onMouseEnter` and `onMouseLeave` view props.

I am uneasy about merging in changes that cannot be upstreamed, which would be the case if we switched to the new API.

Reply:
@NickGerleman - regardless of what happens upstream, we're going to need to have a backward compatibility for `onMouseEnter` / `onMouseLeave` for quite a while, so I don't see why we should block on upstream API changes. Core RN is not concerned (yet) with `onMouseEnter` / `onMouseLeave`, and it shouldn't have to be since it's currently a platform specific API for Web, Windows and macOS.